.github/workflows/publish-python-apidocs.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish Python API Docs
+
+# Run when the Python API changes or every month so that the artifact does not expire
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - onnxruntime_extensions/**
+      - docs/python/**
+  schedule:
+    - cron: '0 0 1 * *'
+  workflow_dispatch:
+
+concurrency:
+  group: "apidocs-python"
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    name: Generate Python API docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install tools
+        run: |
+          sudo apt-get update
+          sudo apt-get install python3-pip
+          sudo apt-get install graphviz
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install -r requirements-dev.txt
+          python3 -m pip install -r requirements.txt
+          python3 -m pip install -e .
+          python3 -m pip list
+      - name: Generate Python docs with Sphinx
+        run: |
+          set -e -x
+          python3 -m pip install -r docs/python/requirements.txt
+          rm -rf sphinx_output/docs/
+          python3 -m sphinx -j1 -v -T -b html -d sphinx_output/docs/extensions/_doctrees/html docs/python/ sphinx_output/docs/extensions/html
```

Comment:
I didn't find where the 'onnxruntime-extensions' package name is specified, did I miss it somewhere?

Reply:
it's in `docs/python/index.rst`